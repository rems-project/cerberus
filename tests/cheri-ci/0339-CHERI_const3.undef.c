int aux(int *p)
{
  return p?(*p = 10):10;
}

int main(void)
{
  const int xxx = (aux(0), xxx);
  aux((int*)&xxx); // the store in aux() is undefined
}