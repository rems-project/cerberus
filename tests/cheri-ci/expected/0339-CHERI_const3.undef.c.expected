Undefined {ub: "UB064_modifying_const", stderr: "", loc: "<3:12--3:21>"}
